content/blog/posts/New-Project-April2017.rst

```diff
@@ -0,0 +1,13 @@
+New Project: New ftp Site for libpng Tarball Distributions
+==========================================================
+:date: 2017-04-28
+:slug: new-project-ftp
+:author: Amanda Kelner
+:tag: featured-stories
+
+The OSL took on a new project and delivers a new ftp:// site for libpng tarball
+distributions that is 20+ times as fast as the old site. It also now responds to
+http:// and https:// requests as well as ftp:// requests. For more information 
```

Comment:
Oh, well that's unfortunate. These shouldn't be clickable links, and didn't realize RST did that by default. Just removing the `://` and capitalizing the protocol acronyms should be sufficient then.

@kelnera sorry =(